Gemfile

```diff
@@ -5,3 +5,4 @@ group :therubyracer do
 end
 
 gem 'github-pages'
+gem 'rake'
```

Comment:
We have a Rakefile in the repo, so it ought to be specified here (and github-pages doesn't bring it in).